pkg/apis/eventing/v1alpha1/eventtype_types_test.go

```diff
@@ -0,0 +1,30 @@
+/*
+Copyright 2019 The Knative Authors
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package v1alpha1
+
+import (
+	"testing"
+)
+
+func TestEventType_GetGroupVersionKind(t *testing.T) {
+	src := EventType{}
+	gvk := src.GetGroupVersionKind()
+
+	if gvk.Kind != "EventType" {
+		t.Errorf("Should be EventType.")
```

Comment:
good job on adding this test case, since it was entirely missing.

IMO this is good enough, since it follows the other patterns (e.g. `CronJobSource{}`)